code/game/objects/items/door_seal.dm

```diff
@@ -0,0 +1,24 @@
+/obj/item/door_seal
+	name = "pneumatic seal"
+	desc = "A brace used to seal and reinforce an airlock. Useful for making areas inaccessible to those without opposable thumbs."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "pneumatic_seal"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+	flags_1 = CONDUCT_1
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+	force = 5
+	throwforce = 5
+	throw_speed = 2
+	throw_range = 1
+	w_class = WEIGHT_CLASS_BULKY
+	custom_materials = list(/datum/material/iron=2000)
```

Comment:
these cost 20000 iron and 10000 plasma to make but you just get 2000 iron?

Reply:
A lot of other stuff seems to contain 10x less materials than it was built with, or leaves it out completely. I left the plasma out because I thought stuff made of plasma explodes, and the plasma was only meant to represent half of it being plasteel anyway. I could up the metal count to like 5000 or something I guess